src/torrent.cpp

```diff
@@ -121,9 +124,12 @@ namespace libtorrent
 		}
 		return ret;
 	}
-
 	constexpr int default_piece_priority = 4;
 
+	// holds seed rank priority duration in the ratio type which is used at
+	// seed rank calculation to prevent multible conversion and constructions
+	static constexpr seconds m_seed_rank_priority_duration = seconds(30 * 60);
```

Comment:
With minutes we would need a conversion on runtime, I guess the compiler is not smart enoght to move an inline cast<seconds>(seed_rank_priority_duration) to a static seconds duration, isn't it?